client/python/easemlclient/easemlclient/model/module.py

```diff
@@ -222,13 +222,13 @@ def __init__(self, id: Optional[List[str]] = None, user: Optional[User] = None,
         if user is not None:
             self._query["user"] = user.id
         if type is not None:
-            self._query["type"] = type.value
+            self._query["type"] = type
```

Comment:
Are we sure this is safe? The intention of `thng.value` was to ensure that when we pass an enum we convert it to string.